packetbeat/magefile.go

```diff
@@ -120,7 +102,7 @@ func Package() {
 
 	devtools.UseElasticBeatPackaging()
```

Comment:
This should now use

https://github.com/elastic/beats/blob/ba2b2f935f1c6badc316f62417d87d630991ad2f/winlogbeat/scripts/mage/package.go#L53

since it's only creating the OSS artifacts.